boards/native/Makefile.include

```diff
@@ -69,7 +69,7 @@ endif
 LINKFLAGS += -ffunction-sections
 
 # set the tap interface for term/valgrind
-ifneq (,$(filter netdev_default,$(USEMODULE)))
+ifneq (,$(filter netdev_tap,$(USEMODULE)))
```

Comment:
IIRC this won't work BTW, since `netdev_tap` might not be selected at this point yet.